examples/basics/Export_V1_to_V2_migration_support.ipynb

```diff
@@ -0,0 +1,869 @@
+{
```

Comment:
<p>Hey! You also said the same thing as this in the title of the next cell. Also, it should be no instead of not. Let's remove one of them since it looks duplicated. "Export V1 methods that will no longer be available after April 20, 2024." Maybe bold this right here, and the title could be Depracted Export V1 Methods or Export V1 Methods. Maybe also include the purpose of the notebook: "Export V1 methods will no longer be available after April 20, 2024. We recommend users to plan accordingly. This notebook is an aid in migrating from Export V1 to Export V2 or something along those lines. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Yea you are totallly right, making updates.</p>